connector_jira/__manifest__.py

```diff
@@ -17,13 +17,13 @@
     ],
     "external_dependencies": {
         "python": [
-            "requests",
-            "jira",
-            "oauthlib",
-            "requests-oauthlib",
-            "requests-toolbelt",
-            "PyJWT",
-            "cryptography",
+            "requests==2.21.0",
```

Comment:
@leemannd As said, try to limit your version pin to the addon that causes the problem.

For instance, you should not pin requests as used in many addons.

Try to find the minimum version of the package you need to pin and use '>=' operator instead.